config/initializers/redis.rb

```diff
@@ -0,0 +1,3 @@
+def redis
```

Comment:
I would not expect to find this here, and it's duplicated with `application.rb` (also an unexpected place for a method like this IMO)

What about something in `app/services` so it's on the autoload path? (I think it would continue to work in `application.rb`)

```ruby
# app/services/redis_service.rb
class RedisService
  def self.redis
    @_redis ||= Redis.new(...)
  end
end
```

Using `RedisService.redis` instead of `redis` everywhere is a little noisier, but clearer and IMO worth it?